README.md

```diff
@@ -12,6 +12,8 @@ Customers will eventually be able to install it using OperatorHub. These instruc
 the latest version of the operator.
 
 ## Pre-requisites
+- OpenShift 4.6+ cluster configured with
+  [hybrid OVN Kubernetes networking](https://github.com/openshift/windows-machine-config-bootstrapper/blob/release-4.4/tools/ansible/docs/ocp-4-4-with-windows-server.md#bring-up-the-openshift-cluster-with-ovn-kubernetes). The linked instructions also apply to 4.6+ clusters.
```

Comment:
I'm worried that `The linked instructions also apply to 4.6+ clusters` will mislead people into thinking all of the instructions listed there will apply to 4.6+ clusters, not just the hybrid OVN instructions.

Reply:
Please add a suggestion as to how you would reword this. `The linked instructions also apply to installing 4.6+ clusters`?

Reply:
Yes i think thats more clear